doc/nrf/releases_and_maturity/migration_guides.rst

```diff
@@ -3,6 +3,22 @@
 Migration guides
 ################
 
+Required changes
+****************
+
+* From this release, Matter over Thread samples work with PSA Crypto API enabled by default instead of mbedTLS as a cryptography backend.
+  To be aware of that change and perform the migration from the mbedTLS PSA Crypto API you need to consider the following points:
+
+  * PSA Crypto API can be disabled by setting the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA` Kconfig option to ``n``.
+    After that, the device will use mbedTLS API as the cryptography backend.
+  * If the DAC (Device Attestation Certificate) private key exists in the factory data set and at least one of the following Kconfig option is set to ``y``:
+
+      * :kconfig:option:`CONFIG_BUILD_WITH_TFM`
+      * :kconfig:option:`CONFIG_SECURE_STORAGE_BACKEND_AEAD_KEY_DERIVE_FROM_HUK`
+
+    it automatically migrates to the PSA ITS secure storage and is replaced by the zeros.
+    To not replace the DAC private key with zeros in the factory data, set the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA_MIGRATE_DAC_PRIV_KEY` Kconfig option to ``n``.
```

Comment:
```suggestion
    To prevent the replacement of the DAC private key with zeros in the factory data, after migration to PSA ITS secure storage, set the :kconfig:option:`CONFIG_CHIP_CRYPTO_PSA_MIGRATE_DAC_PRIV_KEY` Kconfig option to ``n``.
```